runelite-client/src/main/java/net/runelite/client/ui/overlay/components/ImageComponent.java

```diff
@@ -29,16 +29,21 @@
 import java.awt.Point;
 import java.awt.Rectangle;
 import java.awt.image.BufferedImage;
+import lombok.AccessLevel;
 import lombok.Getter;
 import lombok.RequiredArgsConstructor;
 import lombok.Setter;
+import net.runelite.client.util.ImageUtil;
 
 @RequiredArgsConstructor
 @Setter
 public class ImageComponent implements LayoutableRenderableEntity
 {
 	private final BufferedImage image;
 
+	@Setter(AccessLevel.NONE)
```

Comment:
Why did you even added this annotation? you are not using the setter at all

Reply:
To counteract the `@Setter` annotation on line 39

Reply:
Just move the annotation to fields then